src/util/config/build.js

```diff
@@ -25,6 +25,21 @@ export const schema = {
 			arg: 'asset-port',
 			env: process.env.NODE_ENV !== 'test' && 'ASSET_SERVER_PORT', // don't read env in tests
 		},
+		protocol: {
```

Comment:
I think I'm just confused about what goes in which config file although it doesn't need to be settled here.